incubator/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/stream/KafkaGrpcRemoteServerFactory.java

```diff
@@ -176,36 +185,15 @@ public MessageConsumer newStream(
         int length,
         MessageConsumer grpc)
     {
-        return NOOP;
-    }
-
-    private final class RemoteServer
-    {
-        private final Map<OctetsFW, GrpcClient> grpcClients;
-        private final KafkaFetchRemote fetch;
-        private final KafkaGrpcConditionResult result;
-        private final KafkaGrpcFetchHeaderHelper helper;
-
-        private RemoteServer(
-            long originId,
-            long routedId,
-            long entryId,
-            KafkaGrpcConditionResult result,
-            KafkaGrpcFetchHeaderHelper helper)
-        {
-            this.grpcClients = new Object2ObjectHashMap<>();
-            this.fetch = new KafkaFetchRemote(originId, routedId, entryId, result, this);
-            this.result = result;
-            this.helper = helper;
-        }
+        return null;
     }
 
     private final class GrpcClient
```

Comment:
Define `KafkaRemoteServer` and `KafkaCorrelateProxy` before `GrpcClient` in file.